### PR TITLE
Cta box margin

### PR DESCRIPTION
**Pull request by sineade**
:art:

Added class to be able to have left and right margin on outer container around our blue boxes that sit within the grey background.

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/blue-boxes.scss [hunk 1]
```diff
@@ -96,6 +96,10 @@
 }
 
 /*New Grid Block code for blue boxes*/
+.cta-margin{
+  margin: 0 10px 0 10px;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/layout_components/blue-boxes.scss]: use $base-margin

**Comment by sineade** [comment]
:wave: after travis

**Comment by kaytcampbell** [comment]
:art: :art: :art: 
🐘 🐘 🐘

**Comment by kaytcampbell** [comment]
actually :hankey:

**Comment by kaytcampbell** [comment]
you have to bump the version number

**Comment by sineade** [comment]
:wave: after travis for real reals this time

**Comment by kaytcampbell** [comment]
:art: :art: :art:
🐘 🐘 🐘

**Pull request merged**